packages/patternfly-3/patternfly-react/src/components/DualListSelector/DualList.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DualListSelector from './components/DualListSelector';
+import DualListArrows from './components/DualListArrows';
+import { arrangeArray, getCheckedAmount, getCheckedEvent, getDefaultProps, getFilteredResults } from './helpers';
+
+class DualList extends React.Component {
+  constructor(props) {
+    super(props);
+    const defaultProps = getDefaultProps();
+    const left = { ...defaultProps, ...props.left };
+    const right = { ...defaultProps, ...props.right };
+    this.state = {
+      left: {
+        items: arrangeArray(left.items, left.isSortAsc),
+        options: left.options,
+        selectCount: getCheckedAmount(left.items),
+        filterTerm: left.filterTerm,
+        filterResults: left.filterResults,
+        isSortAsc: left.isSortAsc,
+        isMainChecked: left.isMainChecked
+      },
+      right: {
+        items: arrangeArray(right.items, right.isSortAsc),
+        options: right.options,
+        selectCount: getCheckedAmount(right.items),
+        filterTerm: right.filterTerm,
+        filterResults: right.filterResults,
+        isSortAsc: right.isSortAsc,
+        isMainChecked: right.isMainChecked
+      },
+      screenSize: window.innerWidth
+    };
+  }
+
+  componentDidMount() {
+    const { left, right } = this.state;
+    left.isMainChecked && this.onMainCheckboxChange(getCheckedEvent('left'));
+    right.isMainChecked && this.onMainCheckboxChange(getCheckedEvent('right'));
+    window.addEventListener('resize', this.onScreenSize);
```

Comment:
The screen size was used to determine when the arrows between the lists 
should change from left/right to up/down on small screens.

I moved it to css: `transform: rotate(90deg)` instead of reloading the correct Icon,
so now there is no need for this function. :)